### PR TITLE
New mat from point vector

### PR DESCRIPTION
**Pull request by diegohce**
This PR adds a new constructor for the Mat type

### CODE REVIEW

File: core_test.go [hunk 1]
```diff
@@ -3412,3 +3412,23 @@ func TestNewRotatedRect2f(t *testing.T) {
...
+		image.Pt(0, 0), image.Pt(0, size[0]-1),
+		image.Pt(size[1]-1, size[0]-1),
+		image.Pt(size[1]-1, 0),
+	}
+
+	pv := NewPointVectorFromPoints(points)
+	defer pv.Close()
+
+	m := NewMatFromPointVector(pv, false)
+	defer m.Close()
```
Comment (deadprogram) [inline core_test.go:3433]: Maybe add some test that `m` is not empty?

File: core_test.go [hunk 2]
```diff
@@ -3412,3 +3412,23 @@ func TestNewRotatedRect2f(t *testing.T) {
 	}
 
 }
+
+func TestNewMatFromPointVector(t *testing.T) {
+
+	img := IMRead("images/face.jpg", IMReadColor)
```
Comment (deadprogram) [inline core_test.go]: Not sure why you need `img` in this test. You could just set `size` to some specific value.

File: core_test.go (hunk 1 above)
Comment (diegohce) [reply core_test.go:3433]: fixing the test as I write this.

**Comment by diegohce** [comment]
Closed in favor of a cleaner PR.
